src/App.js

```diff
@@ -2,12 +2,24 @@ import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 import { Container } from './components/Container';
 import { Dropdown } from './components/UI/Dropdown';
+import { LANGUAGES } from './constants/language';
+import i18next from 'i18next';
 
 function App() {
+  const handleLanguageChange = (lang = 'en') => {
+    i18next.changeLanguage(lang.name);
+  };
+
   const { t } = useTranslation();
+
   return (
     <ContainerMain>
-      <Dropdown />
+      <Dropdown
+        onSelect={handleLanguageChange}
+        list={LANGUAGES}
+        fontSize={'20px'}
```

Comment:
we shouldn't pass css values as props
if you need a different value for different Dropdowns, just style them
for example
```
const DropdownBig = styled(Dropdown)`
   font-size: 20px;
`
const DropdownSmall = styled(Dropdown)`
   font-size: 12px;
`
```